app/workers/email_worker.rb

```diff
@@ -0,0 +1,7 @@
+class EmailWorker
+  include Sidekiq::Worker
+
+  def perform(email, first_name, title)
+    RentMailer.create(email, first_name, title).deliver
```

Comment:
Here you should use something more generic, suppose you have to send multiples emails types, for example a email for the registration, an email for a payment, an email for a rent, an email for reset password.

Reply:
Done